src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splitArray = sourceString.trim().split(';');
```

Comment:
The variable name 'splitArray' could be more descriptive. Consider naming it to reflect that it contains style declarations, e.g., 'styleDeclarations'.